### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by Krzmbrzl**

### CODE REVIEW

**Comment by Kissaki** [comment]
First two commits look plausible to me.

What’s the reasoning for the other two?

**Comment by Krzmbrzl** [comment]
The rename is to make the name of the CLI executable more unique (it will be needed in the PATH). 
And the last one fixed a build issue for me on Windows. But apparently it broke the CI. I guess I have to find a different solution...

File: CMakeLists.txt [hunk 1]
```diff
@@ -31,6 +31,11 @@ set(JSON_BuildTests OFF CACHE INTERNAL "")
 set(JSON_Install OFF CACHE INTERNAL "")
 set(JSON_ImplicitConversions OFF CACHE INTERNAL "")
 
+if (WIN32)
```
Comment (davidebeatrici) [inline CMakeLists.txt]:
```suggestion
if (static AND WIN32)
```

Comment (davidebeatrici) [reply CMakeLists.txt]: Take my suggestion as a note. It's in case you want to apply the code to the `mumble` repository.

Comment (Krzmbrzl) [reply CMakeLists.txt]: I think I am tending more into the direction of
```suggestion
if (static)
```
and drop the Windows check entirely :thinking:

Comment (davidebeatrici) [reply CMakeLists.txt]: Good idea.

**Comment by Krzmbrzl** [comment]
CI on Azure fails because the respective dir has been removed. But the runs on GitHub actions have all succeeded.

**Pull request merged**